lib/core/DocsLayout.js

```diff
@@ -47,7 +47,8 @@ class DocsLayout extends React.Component {
         description={content.trim().split('\n')[0]}
         language={metadata.language}
         version={metadata.version}
-        metadata={metadata}>
+        metadata={metadata}
+        removeTrailingSlash={true}>
```

Comment:
Would there be time when this would ever be set to `false`? Just wondering if we need a prop at all and we just always remove the trailing slash?